tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -946,12 +955,17 @@ void CopyFusedBatchNormAttributes(const NodeDef& fused_batch_norm,
   (*attr)["is_training"] = src_attr.at("is_training");
   (*attr)["data_format"] = src_attr.at("data_format");
   (*attr)["epsilon"] = src_attr.at("epsilon");
+  (*attr)["exponential_avg_factor"] = src_attr.at("exponential_avg_factor");
 
   // FusedBatchNormV2 and V3 have an extra type parameter.
   if (fused_batch_norm.op() != "FusedBatchNorm") {
     (*attr)["U"] = src_attr.at("U");
   } else {
+#ifndef ENABLE_MKLDNN_V1
     (*attr)["U"] = src_attr.at("T");
+#else
+    SetAttrValue(DT_FLOAT, &(*attr)["U"]);
```

Comment:
Can we set the value the same way as in line 965 to make them look consistent?
```suggestion
    (*attr)["U"] = DT_FLOAT;
```

Reply:
I have changed all of them to use `SetAttrValue`